Two `lanchain-core` tests fail in this fashion:
```
...
        trace_id = posts[0]["trace_id"]
        last_dotted_order = None
        parent_run_id = None
        for name in ordered_names:
            id_ = name_to_body[name]["id"]
            parent_run_id_ = name_to_body[name]["parent_run_id"]
            if parent_run_id_ is not None:
>               assert parent_run_id == parent_run_id_
E               AssertionError: assert '87d11955-4d8...-b1d49926f143' == '270bdab8-581...-cd45954d06bd'
E                 
E                 - 270bdab8-581b-41be-915c-cd45954d06bd
E                 + 87d11955-4d8c-4a97-99d1-b1d49926f143

tests/unit_tests/runnables/test_tracing_interops.py:157: AssertionError
```

Should we skip them ?

--- a/pkgs/development/python-modules/langchain-core/default.nix
+++ b/pkgs/development/python-modules/langchain-core/default.nix
@@ -24,7 +24,7 @@
 
 buildPythonPackage rec {
   pname = "langchain-core";
-  version = "0.2.1";
+  version = "0.2.5";
   pyproject = true;
 
   disabled = pythonOlder "3.8";
@@ -33,7 +33,7 @@ buildPythonPackage rec {
     owner = "langchain-ai";
     repo = "langchain";
     rev = "langchain-core==${version}";
-    hash = "sha256-D0y6kW5bWcCKW2TwVPlZcAUxqADgsOm9fWySAjHYYIg=";
+    hash = "sha256-EBs6BHxBPBR1TfSyesM2gJJtxgClM6lLXko1qisrz7A=";
   };
 
   sourceRoot = "${src.name}/libs/core";
@@ -71,6 +71,11 @@ buildPythonPackage rec {
 
   pytestFlagsArray = [ "tests/unit_tests" ];
 
+  disabledTests = [
+    "test_config_traceable_handoff"
+    "test_config_traceable_async_handoff"
+  ];
+
   passthru = {
     updateScript = writeScript "update.sh" ''
       #!/usr/bin/env nix-shell
@@ -84,11 +89,11 @@ buildPythonPackage rec {
     '';
   };
 
-  meta = with lib; {
+  meta = {
     description = "Building applications with LLMs through composability";
     homepage = "https://github.com/langchain-ai/langchain/tree/master/libs/core";
     changelog = "https://github.com/langchain-ai/langchain/releases/tag/v${version}";
-    license = licenses.mit;
-    maintainers = with maintainers; [ natsukium ];
+    license = lib.licenses.mit;
+    maintainers = with lib.maintainers; [ natsukium ];
   };
 }

--- a/pkgs/development/python-modules/langchain-text-splitters/default.nix
+++ b/pkgs/development/python-modules/langchain-text-splitters/default.nix
@@ -11,7 +11,7 @@
 
 buildPythonPackage rec {
   pname = "langchain-text-splitters";
-  version = "0.2.0";
+  version = "0.2.1";
   pyproject = true;
 
   disabled = pythonOlder "3.9";
@@ -20,7 +20,7 @@ buildPythonPackage rec {
     owner = "langchain-ai";
     repo = "langchain";
     rev = "refs/tags/langchain-text-splitters==${version}";
-    hash = "sha256-QrgZ0j/YdOgEgLeQNSKjOIwqdr/Izuw9Gh6eKQ/00tQ=";
+    hash = "sha256-5le+P+7iGjrTetnTHiJilqtc7G7kJbZnEsUgFyF2aQ8=";
   };
 
   sourceRoot = "${src.name}/libs/text-splitters";
@@ -42,11 +42,11 @@ buildPythonPackage rec {
     inherit (langchain-core) updateScript;
   };
 
-  meta = with lib; {
+  meta = {
     description = "Build context-aware reasoning applications";
     homepage = "https://github.com/langchain-ai/langchain/tree/master/libs/text-splitters";
     changelog = "https://github.com/langchain-ai/langchain/releases/tag/${src.rev}";
-    license = licenses.mit;
-    maintainers = with maintainers; [ fab ];
+    license = lib.licenses.mit;
+    maintainers = with lib.maintainers; [ fab ];
   };
 }

